viz.yaml

```diff
@@ -45,9 +51,49 @@ fetch:
     location: cache/viewbox_limits.rds
     reader: rds
     fetcher: viewbox_limits
-    scripts: [scripts/fetch/viewbox_limits.R]
+    scripts: [scripts/fetch/viewbox_limits.R , scripts/fetch/utils.R]
     depends: ["spatial_metadata", "plot_metadata"]
     comment: sets up the basic view window and projection
+  -
+    id: state_map_data
+    location: cache/state_map_data.rds
+    reader: rds
+    fetcher: map_data
+    fetch_args:
+      database: state
+    scripts: [scripts/fetch/map_data.R , scripts/fetch/utils.R]
+    depends: viewbox_limits
+  -
+    id: county_map_data
+    location: cache/county_map_data.rds
+    reader: rds
+    fetcher: map_data
+    fetch_args:
+      database: county
+    scripts: [scripts/fetch/map_data.R , scripts/fetch/utils.R]
+    depends: viewbox_limits
+  -
+    id: puerto_rico_map_data
+    location: cache/puerto_rico_map_data.rds
+    reader: rds
+    fetcher: map_data
+    fetch_args:
+      database: world
```

Comment:
just a comment, but a higher res map of PR cane come out of `database: mapdata::world2Hires`